src/reusecore/Button/btn.style.js

```diff
@@ -52,7 +52,7 @@ const ButtonStyle = styled.button`
         
     `}
     ${props => props.secondary && css`
-        color: white; 
+        color: black; 
```

Comment:
@ayushthe1 this will have a sweeping affect across all the buttons on the site. We can't do this here.

If the button on the 404 page needs better contrast, you'll want to look to the locally-attached / locally-imported style.js for that section / component instead.

Reply:
thanks for pointing this out @leecalcote  ,i will fix this soon.

Reply:
@leecalcote by isolating out the buttons in the `navigation` section for ci test ,i found that that these passes the `color-contrast` test ,so i'm setting back the color to white and no additional changes are required for this.

Reply:
Oh, I see. Makes sense. A necessary test.